packages/core/src/codewhisperer/service/transformByQ/transformationResultsViewProvider.ts

```diff
@@ -402,6 +403,37 @@ export class ProposedTransformationExplorer {
                     `${CodeWhispererConstants.errorDeserializingDiffNotification} ${deserializeErrorMessage}`
                 )
             }
+
+            try {
+                const metricsPath = path.join(pathContainingArchive, ExportResultArchiveStructure.PathToMetrics)
+                const metricsData = JSON.parse(fs.readFileSync(metricsPath, 'utf8'))
+
+                console.log('metricsData.linesOfCodeChanged =', metricsData.linesOfCodeChanged)
+                console.log('metricsData.charactersOfCodeChanged =', metricsData.charactersOfCodeChanged)
+                console.log('linesOfCodeSubmitted =', transformByQState.getLinesOfCodeSubmitted())
+                console.log('transformByQState.getTransformationType() =', transformByQState.getTransformationType())
+
+                await codeWhisperer.codeWhispererClient.sendTelemetryEvent({
+                    telemetryEvent: {
+                        transformEvent: {
+                            jobId: transformByQState.getJobId(),
+                            timestamp: new Date(),
+                            ideCategory: 'VSCODE',
```

Comment:
Is there any constant value we can extract this `VSCODE` value from? Same for `JAVA` and `SQL` below

Reply:
+1, do we have enum or constants represent IDE or get it from IDE env ?

Reply:
I don't see this no. All other uses in toolkit seem to use `VSCODE` just like this